README.md

```diff
@@ -155,6 +155,22 @@ gke-on-prem-cluster-f1-pvc-bcc115d6-6472-11e8-a9b6-42010a800140  us-west1  regio
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```

Comment:
`[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

Reply:
please test this